build.gradle

```diff
@@ -7,8 +7,10 @@ buildscript {
 		}
 	}
 	dependencies {
-		classpath "cuchaz:enigma:0.15.1+build.164"
-		classpath "net.fabricmc:stitch:0.4.5+build.73"
+		classpath "cuchaz:enigma:0.15.2+build.165"
+		classpath("net.fabricmc:stitch:0.4.5+build.73") {
+			exclude group: "cuchaz", module: "enigma"
```

Comment:
do we actaully need this?

Reply:
Turns out if the enigma above fails to resolve, gradle will auto fall back to that one with stitch, which happened in that last #1287 pr